pkg/inference/scheduling/scheduler.go

```diff
@@ -300,3 +300,8 @@ func (s *Scheduler) ConfigureRunner(ctx context.Context, backend inference.Backe
 
 	return backend, nil
 }
+
+// GetAllModelConfigs retrieves all active model configurations.
+func (s *Scheduler) GetAllModelConfigs(ctx context.Context) []ModelConfigEntry {
```

Comment:
nit: I think the handler has access to the loader via scheduler:
`h.scheduler.loader` so we might used directly and this method would not be needed